docs/benefits_of_contextualized_ML.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>We should explain that we believe X is generated by a network, which has parameters dependent on C. Based on this believe, we can  then factorize P(C, X). The way it's phrased right now makes it seem like the BN formulation is appropriate for any latent model, but this is a design choice that only works when we <em>believe</em> the latent model is a BN.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>